### ISSUE TITLE
"Multi touch" functionality

### ISSUE THREAD

**Issue by averageapps** [MohammadAG/Xposed-Knock-Code#2]
I think It would be nice if the provided lock screen behaved a bit more like the stock PIN unlock method in the way that taps are registered:
You don't have to touch and release every field before touching another field, because only the start of the touch (I think it is MotionEvent.ACTION_DOWN) is considered when checking if it was the right field. This allows quicker unlocking, because you don't have make sure that only one finger is touching the screen at a time.

Could this behavior be implemented by some small modifications in lines 151 - 183 in KnockCodeView.java? (I can't test it myself right now.)

**Comment by MohammadAG** [comment]
Wouldn't this cause more accidental presses? It would also make tap and hold harder to process. Right now it's used for clearing input.

**Comment by averageapps** [comment]
I do not think that it would lead to more accidental presses, rather the opposite.
Maybe my description was a bit unclear, so I will provide some examples:
The behavior that I suggest can be found in several similar systems in Android, for example in the PIN unlock method, the stock phone dialer app and the stock keyboard (try pressing one number / key and then another one while still touching the first one to see what I mean).
So I thought this is the expected behavior.

**Comment by till-robert** [comment]
@acerageapps i agree with you. If you want to type in the code with both thunbs it can easily happen that you habe both fingers in the screen... Then the app doesn't recognize the second tap.
